src/components/App.tsx

```diff
@@ -2075,9 +2075,19 @@ class App extends React.Component<ExcalidrawProps, AppState> {
             document.documentElement.style.cursor = CURSOR_TYPE.POINTER;
           }
           // update last uncommitted point
+          const lastUncommittedPoint = () => {
```

Comment:
I prefer making it a ternary instead of a function, but it might be just a preference.

Reply:
Happy to change to ternary if needed...I personally preferred the function though:)

Reply:
Ternary added